src/main/java/uk/ac/ebi/embl/converter/Main.java

```diff
@@ -0,0 +1,37 @@
+package uk.ac.ebi.embl.converter;
+
+import java.io.BufferedReader;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import uk.ac.ebi.embl.converter.gff3.GFF3File;
+import uk.ac.ebi.embl.converter.fftogff3.FFGFF3FileFactory;
+import uk.ac.ebi.embl.flatfile.reader.ReaderOptions;
+import uk.ac.ebi.embl.flatfile.reader.embl.EmblEntryReader;
+import uk.ac.ebi.embl.api.validation.ValidationResult;
+import uk.ac.ebi.embl.api.entry.Entry;
+
+public class Main {
+
+  public static void main(String[] args) throws Exception {
+    String filename = "src/test/resources/FFFeaturesToGFF3Features/gene_mrna_parents.embl";
```

Comment:
We might need to think about in/out arguments to this application.
input 
output